app/controllers/v0/notifications_controller.rb

```diff
@@ -27,6 +27,16 @@ def show
       end
     end
 
+    def update
+      validate_record_present!(@notification, subject)
+
+      if @notification.update(read_at: read_at)
```

Comment:
is there a way to update status?

Reply:
🤔 It was my understanding from our discussions that we would be:

- using the `dismissed_statuses_controller.rb` for any `status` and `status_effective_at` related updates
- using the `notifications_controller.rb` for any vanilla "read/unread" notification use cases
- using the `Notification` db table to handle both use cases, and the pattern for those use cases would be presented through the two dedicated controllers

No?